committeeoversightapp/models.py

```diff
@@ -9,17 +19,108 @@ class HearingCategoryType(models.Model):
     def __str__(self):
         return u'({0}) {1}'.format(self.id, self.name)
 
+
 class HearingCategory(models.Model):
     event = models.ForeignKey(Event, null=True, on_delete=models.CASCADE)
-    category = models.ForeignKey(HearingCategoryType, null=True, on_delete=models.CASCADE)
+    category = models.ForeignKey(HearingCategoryType,
+                                 null=True,
+                                 on_delete=models.CASCADE)
+
 
 class WitnessDetails(models.Model):
     witness = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
-    document = models.ForeignKey(EventDocument, null=True, blank=True, on_delete=models.CASCADE)
-    organization = models.CharField(max_length=100, null=True, blank=True, primary_key=False)
+    document = models.ForeignKey(EventDocument,
+                                 null=True,
+                                 blank=True,
+                                 on_delete=models.CASCADE)
+    organization = models.CharField(max_length=100,
+                                    null=True,
+                                    blank=True,
+                                    primary_key=False)
     retired = models.BooleanField(default=False)
 
+
 class Committee(models.Model):
     lugar_id = models.IntegerField(null=True, blank=True, primary_key=False)
-    lugar_name = models.CharField(max_length=200, null=True, blank=True, primary_key=False)
-    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.CASCADE)
+    lugar_name = models.CharField(max_length=200,
+                                  null=True,
+                                  blank=True,
+                                  primary_key=False)
+    organization = models.ForeignKey(Organization,
+                                     null=True,
+                                     blank=True,
+                                     on_delete=models.CASCADE)
+
+
+class StaticPage(Page):
+    body = StreamField([
+        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
+        ('paragraph', blocks.RichTextBlock()),
+        ('image', ImageChooserBlock()),
+        ('button',
+         blocks.StructBlock([
+            ('button_text', blocks.CharBlock()),
+            ('button_link', blocks.URLBlock())
+            ],
+         icon='site'))
+    ])
+
+    # Editor configuration
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('body'),
+    ]
+
+
+class LandingPage(Page):
```

Comment:
what is the difference between a static page and a landing page?

Reply:
The landing page needs a table with all the committees and their ratings, per the wireframes. Static pages correspond to about, methodology, and contact in the wireframes.

Reply:
got it! is creating two models what allows us to define two separate templates for the different page types?

Reply:
Yep! Each model in Wagtail corresponds to a template.

Reply:
right on! would you mind leaving a comment to the effect of "define static and landing pages as separate models in order to define separate templates" on one or both of these models? it wasn't immediately clear to me why we'd need what appeared to be a duplicate model.